packages/rocketchat-statistics/server/functions/get.js

```diff
@@ -59,6 +59,11 @@ RocketChat.statistics.get = function _getStatistics() {
 		uptime: process.uptime()
 	};
 
+	statistics.deploy = {
+		method: process.env.DEPLOY_METHOD || 'tar',
+		platform: process.env.DEPLOY_PLATFORM || 'selfinstall'
```

Comment:
problem with that is we have to continue to update that list, any time we or someone else adds a new one. :frowning_face: